example/src/test/kotlin/pl/touk/krush/BookTest.kt

```diff
@@ -18,13 +18,16 @@ class BookTest : BaseDatabaseTest() {
 
             // given
             val book = Book(
-                    isbn = "1449373321", publishDate = LocalDate.of(2017, APRIL, 11),
-                    title = "Designing Data-Intensive Applications", author = "Martin Kleppmann"
+                    isbn = "1449373321",
+                    publishDate = LocalDate.of(2017, APRIL, 11),
```

Comment:
\[detekt\] INFO: \[style/MagicNumber\] Report magic numbers\. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is\. It's better to declare such numbers as constants and give them a proper name\. By default, \-1, 0, 1, and 2 are not considered to be magic numbers\.